src/Navbar.test.js

```diff
@@ -0,0 +1,31 @@
+import Navbar from "./Navbar";
+import { configure, shallow } from "enzyme";
+import Adapter from "enzyme-adapter-react-16";
+import { NavLink } from 'react-router-dom';
+import React from "react";
+
+configure({ adapter: new Adapter() });
+//adapter adapts enzyme for our version of react?
+
```

Comment:
Yeah so the adapter is used to create a bridge between the enzyme API which I don't think has really changed and the forever changing versions of react, with different versions of react they release different render methods and different ways to interact with the DOM which would interfere with the `shallow` and `mount` , so the adapters allow you to write tests normally depending on which version of react

Reply:
Perfect, that makes sense! Thank you